platform/linux-generic/Makefile.am

```diff
@@ -94,6 +96,8 @@ noinst_HEADERS = \
 		  include/odp_forward_typedefs_internal.h \
 		  include/odp_internal.h \
 		  include/odp_ipsec_internal.h \
+		  include/odp_libconfig_config.h \
+		  include/odp_libconfig_internal.h \
```

Comment:
Last line should go to `nodist_noinst_HEADERS`, so it does not get included into distribution.

Reply:
OK

Reply:
I assume you meant odp_libconfig_config.h  Anyway, doing this breaks distcheck build:

```
  CC       odp_libconfig.lo
../../../../platform/linux-generic/odp_libconfig.c:17:34: fatal error: odp_libconfig_config.h: No such file or directory
compilation terminated.
Makefile:1120: recipe for target 'odp_libconfig.lo' failed
make[2]: *** [odp_libconfig.lo] Error 1
make[2]: Leaving directory '/home/nnn/dev/odp.git/opendataplane-1.17.0.0/_build/sub/platform/linux-generic'
```

With both headers in noinst_HEADERS distcheck completes but complains at the end:
```
ERROR: files left in build directory after distclean:
./platform/linux-generic/include/odp_libconfig_config.h
Makefile:793: recipe for target 'distcleancheck' failed
make[1]: *** [distcleancheck] Error 1
make[1]: Leaving directory '/home/nnn/dev/odp.git/opendataplane-1.17.0.0/_build/sub'
Makefile:719: recipe for target 'distcheck' failed
make: *** [distcheck] Error 1
```

Any pointers?

Reply:
Adding `DISTCLEANFILES = include/odp_libconfig_config.h` fixed the discheck failure when odp_libconfig_config.h is added to noinst_HEADERS.